cmd/procs/execution/executioner.go

```diff
@@ -47,6 +47,10 @@ func NewCmd(printer io.Printer, proctorEngineClient daemon.Client) *cobra.Comman
 
 			executedProcName, err := proctorEngineClient.ExecuteProc(procName, procArgs)
 			if err != nil {
+				if err.Error() == "Unauthorized" {
```

Comment:
Please extract "Unauthorized" as a utility error/constant because it's used in multiple places.